server/src/helpers/sendEmail.ts

```diff
@@ -0,0 +1,36 @@
+import nodemailer from 'nodemailer';
+
+const sendEmail = (userEmail: string, username: string, jobTitle: string) => {
+  const transporter = nodemailer.createTransport({
+    host: 'smtp.gmail.com',
+    port: 587,
+    secure: false,
+    auth: {
```

Comment:
```js
auth: {
      user: proccess.env.APP_EMAIL,
      pass: proccess.env.APP_EMAIL_PASSWORD,
    },
```
I think its will be more safe to put the email in the env file

Reply:
💯